### DISCUSSION TITLE
Vercel Custom Domain Linkedin sharing issue

### DISCUSSION THREAD

**Discussion by oshanfernando** [vercel/next.js#13707] [Help] [answered]
So i hosted my website on vercel and added a custom domain to it and its working fine. When i try to share a post from that domain, linkedin blocks the request with a message that it's suspected as a malicious website. But when i share from the automatically assigned deployment URLs, it works fine.
Has anyone come across this issue before and if so, it is because of the default SSL certificates attached by vercel ?

FYI, i checked the domain for any malware and none were reported.

**Comment by paulogdm** [comment, marked as answer]
Hello @oshanfernando ,

This is indeed a problem with our CDN that affects specific sites on our platform. Our team is tracking the issue, and I will get back to you once this is fully resolved.

**Reply by oshanfernando** [reply]
Thank you @paulogdm. [this](https://www.codingoutpost.com) is my domain if it is of any help.

**Reply by oshanfernando** [reply]
@paulogdm do you have any updates on this ? Seems like the issue still persists.

**Reply by paulogdm** [reply]
Hello @oshanfernando ,

We just fixed this issue a few hours ago. Everything should work as expected.

**Reply by oshanfernando** [reply]
Thank you @paulogdm

**Reply by webJdi** [reply]
Hey @paulogdm!
I'm facing similar issue with my vercel app: https://learnbuddy-gamma.vercel.app/

Could you hell me out as well?

**Comment by AroniasPrenovost** [comment]
I'm experiencing something similar, where my LinkedIn posts that reference my vercel-hosted app show "404, page not found")

**Comment by paulogdm** [comment]
Locked as this was a very specific case at the time it was first reported.
For further issues, please create a ticket with Vercel: https://vercel.com/docs/dashboard-features/support-center
cc @AroniasPrenovost @webJdi
